src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparatorBlockCommentSeparatedFromPackage.java

```diff
@@ -0,0 +1,12 @@
+/*
+* Config = default
```

Comment:
https://checkstyle.org/config_whitespace.html#EmptyLineSeparator_Description
please reference all properties. 
example: https://github.com/checkstyle/checkstyle/blob/master/src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoccontentlocation/InputJavadocContentLocationTrailingSpace.java#L4

it helps a lot.

Reply:
@romani I added properties and changed attention note, pls check if you are ok with it